map.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Bushfire Intensity</title>
+
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.3/dist/leaflet.css" integrity="sha512-Rksm5RenBEKSKFjgI3a41vrjkw4EVPlJ3+OiI65vTjIdo9brlAacEuKOiQ5OFh7cOI1bkDwLqdLw3Zg0cRJAAQ==" crossorigin="" />
+    <script src="https://unpkg.com/leaflet@1.3.3/dist/leaflet.js" integrity="sha512-tAGcCfR4Sc5ZP5ZoVz0quoZDYX5aCtEm/eu1KhSLj2c9eFrylXZknQYmxUssFaVJKvvc0dJQixhGjG2yXWiV9Q==" crossorigin=""></script>
+    <script src="https://d3js.org/d3.v5.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
Is this link to style.css meant to be a link to a generic style sheet or link to a style sheet specific to the map component? If it's specific to the map component, maybe change the name to mapStyles.css. If it's a generic stylesheet for the entire app and to avoid duplicating this css link in all of the html files, i think it would be a good idea to have like a layout/generic html template that contains all of the generic css stylesheets and js scripts that are used across the app all in one place. This will all need to (eventually) move to the flask app. So fine for now until we need to move it.

Reply:
sounds good! happy to have the map-specific css live inside a larger css file (if that doesn't conflict with best practice).